lib/service/desktop.go

```diff
@@ -101,8 +101,8 @@ func (process *TeleportProcess) initWindowsDesktopServiceRegistered(log *logrus.
 	case useTunnel && !cfg.WindowsDesktop.ListenAddr.IsEmpty():
 		return trace.BadParameter("either set windows_desktop_service.listen_addr if this process can be reached from a teleport proxy or point teleport.auth_servers to a proxy to dial out, but don't set both")
 	case !useTunnel && cfg.WindowsDesktop.ListenAddr.IsEmpty():
-		return trace.BadParameter("set windows_desktop_service.listen_addr if this process can be reached from a teleport proxy or point teleport.auth_servers to a proxy to dial out")
-
+		cfg.WindowsDesktop.ListenAddr = *defaults.WindowsDesktopProxyListenAddr()
+		fallthrough
```

Comment:
Cool, I wasn't aware of this feature

Reply:
I think there are some docs that need updating if this is the case.

This only works when Windows Desktop Service is running in the same process as the Teleport Proxy Service, right?

Reply:
I was just commenting on the `fallthrough` keyword.

I'm not actually sure, I'm pretty ignorant about the multiplex proxy setup, and was just reviewing the code logic itself.